src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx

```diff
@@ -143,6 +143,8 @@ type RSSFeedsFormProps = {
     discovered: 'allBlogs' | 'allChannels';
   };
   prompt: string;
+  promptExample1: string;
+  promptExample2: string;
```

Comment:
Instead of `promptExample1` and `promptExample2`, won't it be better to take an array with prop example?